ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -908,6 +929,10 @@ - (Class)questionResultClass {
     return [ORKBooleanQuestionResult class];
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    return [self.impliedAnswerFormat stringForAnswer: @[answer]];
+}
```

Comment:
I was wrong. We need to wrap the boolean `answer` into an array and pass that array to the implied answer format (text choice).